lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -289,6 +293,10 @@ function REPL( options ) {
 	* @param {Object} key - key object
 	*/
 	function beforeKeypress( data, key ) {
+		if ( key && key.name === 'tab' ) {
+			completerEngine.beforeKeypress();
+			return;
```

Comment:
Does this not conflict with the preview completer also using the TAB character to complete the current line? L304.

Reply:
The preview completer only uses right arrow and ENTER for auto completing, tab completion was supported by the readline's completer itself

Reply:
Thanks for clarifying.